samples/ServiceDefaults/Extensions.cs

```diff
@@ -126,13 +130,13 @@ public static WebApplication MapDefaultEndpoints(this WebApplication app)
         if (app.Environment.IsDevelopment())
         {
             // All health checks must pass for app to be considered ready to accept traffic after starting
-            app.MapHealthChecks("/health");
+            app.MapHealthChecks("/health").ShortCircuit();
```

Comment:
Might be a dumb question but won't this be already part of the above call to configure http clients where we add the standard resilience handler?